src/constructors.jl

```diff
@@ -34,7 +34,7 @@ for TM in tupleTMs
 
         # Short-cut for independent variable
         $(TM)(ord::Integer, x0, dom::Interval{T}) where {T} =
-            $(TM)(interval(x0) + Taylor1(Interval{T}, ord), zero(dom), x0, dom)
+            $(TM)(interval(x0) + Taylor1(Interval{T}, ord), zero(dom), interval(x0), dom)
```

Comment:
It just occurred to me that we could make this method a bit more general, e.g., to return a TaylorModel with `Float64` polynomial coefficients. Then, this method would need to be:
```julia
$(TM)(ord::Integer, x0::T, dom::Interval{T}) where {T} =
    $(TM)(x0 + Taylor1(T, ord), zero(dom), interval(x0), dom)
```
Let me know if you want to do this, or if I simply go ahead and merge (and latter we do the change).